fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/Sequence.java

```diff
@@ -37,7 +38,12 @@ public void apply(Integer priorVersion, IDatabaseAdapter target) {
         if (priorVersion != null && priorVersion > 0 && this.version > priorVersion) {
             throw new UnsupportedOperationException("Upgrading sequences is not supported");
         }
-        apply(target);
+
+        // Only if VERSION1 then we want to apply, else fall through
+        // Re-creating a sequence can have unintended consequences.
+        if (this.version == 1 && priorVersion == 0) {
+            apply(target);
+        }
```

Comment:
this skips the apply when priorVersion == null, which is what that `AllVersionHistoryService` service always passes.  I see we don't use that any more, but i think we should also handle nulls here unless we totally remove it

Reply:
yes - I accepted your change - a good guard is now in place.